HealthcareSystem/EventSourcingService/Model/StatisticEvent.cs

```diff
@@ -0,0 +1,27 @@
+﻿using EventSourcingService.Model.Enum;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EventSourcingService.Model
+{
+    public class StatisticEvent : DomainEvent
```

Comment:
I would probably rename the event, just because all the events will be used for statistics of some sort so it won't be very descriptive. Maybe something like PatientSchedulingEvent?